edx-olive/edx-platform
# SE-3494 Revert "hide ccx courses from catalog"

This reverts commit b44713f2957d68dac257fcb99984d27164a74bf2.

This is no longer required for client, and by reverting this, we reduce
code drift from upstream.

**Test instructions**:

- verify that this reverts https://github.com/edx-olive/edx-platform/pull/6 exactly
- deploy to stage and do some spot checks viewing courses you do and don't have access to (the code change was minor and clean, so it should be fine)

**Reviewers**:
